CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Eclipse GLSP Server Node Changelog
 
+## v1.1.0 - upcoming
+
+### Breaking Changes
+
+-   [graph] Align GGraph model with newest changes from glsp-server [#22](https://github.com/eclipse-glsp/glsp-server-node/pull/22) - Contributed on behalf of STMicroelectronics
+    -   Renamed interfaces:
+        -   `EdgePlacement` -> `GEdgePlacement` (affected classes: `GEdgeLayoutable`, `GLabel`)
+        -   `GLayoutContainer` (`glayout-container.ts`) -> `GLayouting` (`glayouting.ts`) (affected classes: `GCompartment`, `GGraph`, `GNode`)
+        -   `GShapePreRenderedElement` (`gpre-shape-prerendered-element.ts`) -> `GShapedPreRenderedElement` (`gshaped-prerendered-element.ts`)
+    -   Renamed file names: `gbound-aware.ts` -> `gbounds-aware.ts`
+    -   `GGraph` now additionally extends `GLayoutable`
```

Comment:
API additions don' need to be mentioned in the breaking change section (as they don't break any existing API :wink:)
Please remove the last three lines